openbb_platform/core/openbb_core/app/command_runner.py

```diff
@@ -237,23 +237,20 @@ def _chart(
                 raise OpenBBError(
                     "Charting is not installed. Please install `openbb-charting`."
                 )
+            # Here we will pop the chart_params kwargs and flatten them into the kwargs.
             chart_params = {}
-            extra_params = kwargs.get("extra_params", {})
+            extra_params = getattr(obbject, "_extra_params", {})
```

Comment:
I don't think this is accurate: the `OBBject` doesn't have this attribute.

Reply:
I see, it is injected on the function execution. But maybe we should add that attribute to the `OBBject` then.